updater/lib/dependabot/updater/operations/update_all_versions.rb

```diff
@@ -52,11 +52,16 @@ def perform
           Dependabot.logger.info("Starting update job for #{job.source.repo}")
           Dependabot.logger.info("Checking all dependencies for version updates...")
 
+          # Raise an error if the package manager version is unsupported
+          dependency_snapshot.package_manager&.raise_if_unsupported!
```

Comment:
When we have `unsupported error` for the package manager, we don't want to go over each dependency, instead we want to raise unsupported error for the whole process and handle error properly.